GitHubCore.ps1

```diff
@@ -265,8 +272,14 @@ function Invoke-GHRestMethod
 
                 try
                 {
+                    # Temporarily Disable Progress Bar during Invoke-WebRequest
+                    $tempProgressPreference = $ProgressPreference
```

Comment:
If we're going to restore it, we should save the current value _before_ the `try`, and restore it in a `finally`.  I'm fine if we don't save/restore in this scenario (as long as we leave a comment when we're doing it), since this is happening in a separate PowerShell session that will be disappearing as soon as this job is collected.

Reply:
It is all a bit academic anyway, as the `$ProgressPreference` variable is scoped to the function, so  I've removed the saving and restoring of its value.

Reply:
Ah, that's interesting (and confusing).  I had assumed that was a global var state and any change affected global state.  I just verified that you're right...changes made to it within the context of a function aren't retained outside of that function's scope:

```
function Set-Pref {
    Write-Host ": [$ProgressPreference]"
    $ProgressPreference = 'Silent'
    Write-Host ": [$ProgressPreference]"
}

Write-Host "($ProgressPreference)"
Set-Pref
Write-Host "($ProgressPreference)"
$ProgressPreference = 'SilentlyContinue'
Write-Host "($ProgressPreference)"
Set-Pref
Write-Host "($ProgressPreference)"
```

For the future me (or someone else) who looks at the code later, can you make a further change to that comment and indicate that the change will not be retained outside of that function scope?

Reply:
Done.